framework/src/modules/chain/transactions/votes_weight.js

```diff
@@ -121,7 +123,8 @@ const updateSenderDelegateVotes = (
 	exceptions,
 	undo = false,
 ) => {
-	const amount = transaction.fee.plus(transaction.amount);
+	// use the ammount or default to zero as LIP-0012 removes the 'amount' property from all transactions but transfer
+	const amount = transaction.fee.plus(transaction.asset.amount || '0');
```

Comment:
in every other place, I see default amount is integer zero and here its string zero, why?.

Reply:
Typo. Fixed. By the way this is to default to zero instead of having an `if` everywhere in the code to account for the transfer transaction now being the only one with the amount property. Fixed on https://github.com/LiskHQ/lisk-sdk/pull/4390